### PR TITLE
update tutorial for 3.0

### LINKED ISSUE #12821: Adding Client-side library doesn't work in Visual Studio 2019 - no autocomplete displayed
This documentation says an autocomplete of available client-side javascript libraries will be displayed. But it doesn't work. So its not possible to see what javascript signalr client versions are available.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0dd2991d-c22d-ad0f-80b6-bd1b503db8f3
* Version Independent ID: 5e3fbe1d-c97f-9940-4f0c-067d4b2112b1
* Content: [Get started with ASP.NET Core SignalR](https://docs.microsoft.com/en-us/aspnet/core/tutorials/signalr?view=aspnetcore-2.2&tabs=visual-studio#feedback)
* Content Source: [aspnetcore/tutorials/signalr.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/signalr.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @bradygaster
* Microsoft Alias: **bradyg**

### PR DESCRIPTION
**Pull request by bradygaster**
fixes #12821 by documenting the flow to add the newly-located `@microsoft/signalr` package. **Note** - I only updated the content in the 3.0 moniker range and left 2.2 as-it-was.

### CODE REVIEW

File: aspnetcore/tutorials/signalr.md [hunk 1]
```diff
@@ -92,16 +92,16 @@ The SignalR server library is included in the ASP.NET Core 3.0 shared framework.
 
 * In the **Add Client-Side Library** dialog, for **Provider** select **unpkg**.
 
-* For **Library**, enter `@aspnet/signalr@next`.
+* For **Library**, enter `@microsoft/signalr@next`.
```
Comment (BrennanConroy) [inline aspnetcore/tutorials/signalr.md]: change to latest and remove the comment below :D

**Comment by Rick-Anderson** [comment]
cc @wadepickett

**Comment by Rick-Anderson** [comment]
@bradygaster  @BrennanConroy  doesn't the code need to be updated for the 3.0 templates?

The 3.0 templates don't generate `CookiePolicyOptions`.

```
 services.Configure<CookiePolicyOptions>(options =>
            {
                // This lambda determines whether user consent for non-essential cookies is needed for a given request.
                options.CheckConsentNeeded = context => true;
            });
```

**Comment by bradygaster** [comment]
@Rick-Anderson and @wadepickett  - my fix was **specific** to the issues a customer reported with installing the client libraries for SignalR. I hadn't gone through the rest of it yet, but that's great feedback and I agree it should be updated. if you were working on that already is there a chance our PRs could be used together for a full-stop solution?

**Comment by scottaddie** [comment]
I'll take care of incorporating Rick's suggestion since Brady is traveling. Look for a separate PR with those changes.

**Comment by wadepickett** [comment]
@scottaddie , I already had Rick's suggestions fixed and others in a duplicate issue #14755.  If OK with you I will add it.  At least I get to use some of my work from the dupe.  Too late?

**Comment by wadepickett** [comment]
Per conversation with scottaddie I will  put remaining updates in a new pr.

**Comment by wadepickett** [comment]
@bradygaster, I am adding some updates to this tutorial and will put you on review in a new pr.
For what was checked in on this one I do have a questions:   The .md instructions say: For Library, enter "@microsoft/signalr@latest" but the libman images shows instead the specific version signalr@3.0.0.    I had assumed we should use @latest.  Which should I go with going forward?

**Comment by bradygaster** [comment]
go with **@latest** moving forward.

**Comment by wadepickett** [comment]
@bradygaster, that minor @latest screenshot update was taken care of in #14929 today.

**Pull request merged**